### DISCUSSION TITLE
Permission issues at setting up tokenizer

### DISCUSSION THREAD

**Discussion by gsg-git** [osm-search/Nominatim#3194] [Q&A] [answered]
Hello!
I have the same eror like in this discussion: https://github.com/osm-search/Nominatim/discussions/2549#discussion-3733313
Sadly the solution is missing.

`PermissionError: [Errno 13] Permission denied: '/home/nominatim-planet/tokenizer'`

Im thankful for any hint!

Thank you in advance!

**Comment by mtmail** [comment]
Please add more information. What are the permission flags on the directory, parent directory and files. Which user name and group are you using to access the directory? How did you install Nominatim, which version of Nominatim are we talking about, what operating system.

**Reply by gsg-git** [reply]
The System is Ubuntu 22.04.3 LTS Server.
I use the default documentation: https://nominatim.org/release-docs/latest/admin/Installation/ and try to import the fles from the project folder with the default user.
The permissions of the folder is:
`drwxr-xr-x  2 root root 4096 Sep  8 12:48 nominatim-planet`

i build it from the default downoad (4.3.0) without Docker.

**Reply by lonvia** [reply, marked as answer]
The project directory must not belong to root but to the user you are running the import with.

**Reply by gsg-git** [reply]
so i use:
`sudo setfacl -R -m u:gsa:rwx /home/nominatim-planet`
For the project folder (gsa iss the default user)?

**Reply by gsg-git** [reply]
Uptil now everyting looks very fine!

**Reply by gsg-git** [reply]
Wie lange dauert es ca für nur Deutschland? läuft auf einer 2TB  Sata SSD mit 550 MB/s ?

**Reply by gsg-git** [reply]
@lonvia @mtmail Es läuft - vielen Dank!
